Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Ammunition/Magazines/M41-A_mag.yml

```
@@ -1,90 +1,90 @@
-  name: "magazine (.20 rifle uranium)"
-  parent: LPPBaseMagazineRifleM41A
-  components:
-  - type: BallisticAmmoProvider
-    proto: CartridgeRifleUranium
-  - type: Sprite
-    layers:
-    - state: uranium
-      map: ["enum.GunVisualLayers.Base"]
-    - state: mag-1
-      map: ["enum.GunVisualLayers.Mag"]
+- type: entity
```

yamllint

[error] wrong new line character: expected \n

(new-lines)